requirements.txt

```diff
@@ -1,8 +1,9 @@
 # api
-python-decouple
 django
-restless
 psycopg2
+python-Levenshtein
```

Comment:
this should be fixed, since we did it on the last 2 PRs

Reply:
True story! Fixed in f9ea49a.